File: test/func_examples/function.go
package main

import (
	"encoding/json"
	"fmt"
)

func main() {
	// GET /test/result/golang_string
	fmt.Println("Hello, Golang!")

	// GET /test/result/golang.json
	greet := map[string]string{"hello": "golang"}
	j, _ := json.Marshal(greet)
	fmt.Println(string(j))
}

File: test/func_examples/function.py
import json
# import but not used, just for testing package installation
import openpyxl

# GET /test/result/python.json

greeting = {"hello": "world"}
print(json.dumps(greeting))

# GET /test/result/python_second.json

print(json.dumps(greeting))

# GET /test/result/python_string

print("hello world")
